### PR TITLE
added 7.3

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: config/projects.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 admin-bundle:
   branches:
     master:
-      php: ['7.2']
+      php: ['7.3', '7.2']
```
Comment (OskarStark) [inline config/projects.yml]: @greg0ire is this order correct? I want to use 7.3 for lint and docs

Comment (greg0ire) [reply config/projects.yml]: I really don't know, sorry.

Comment (Soullivaneuh) [reply config/projects.yml]: Please reverse, 7.2 first.

File: config/projects.yml [hunk 2]
```diff
@@ -18,13 +18,13 @@ admin-bundle:
 admin-search-bundle:
   branches:
     master:
-      php: ['7.2']
+      php: ['7.3', '7.2']
```
Comment (Soullivaneuh) [inline config/projects.yml]: Reverse.

File: config/projects.yml [hunk 3]
```diff
@@ -33,25 +33,25 @@ admin-search-bundle:
 article-bundle:
   branches:
     master:
-      php: ['7.2']
+      php: ['7.3', '7.2']
```
Comment (Soullivaneuh) [inline config/projects.yml]: Reverse, and the other. :arrow_double_down:

**Comment by OskarStark** [comment]
Done, please review again and merge!

**Pull request merged**